app/models/backend_api_config.rb

```diff
@@ -13,7 +13,10 @@ class BackendApiConfig < ApplicationRecord
 
   validates :path, length: { in: 0..255, allow_nil: false }, path: true
 
-  scope :with_subpath, -> { common_query = where.not(path: '/'); System::Database.oracle? ? common_query.where('path is NOT NULL') : common_query.where.not(path: '') }
+  scope :with_subpath, (lambda do
```

Comment:
Do you need parenthesis here?

Reply:
The parenthesis are just for Rubocop
![image](https://user-images.githubusercontent.com/11318903/65391322-378f8280-dd68-11e9-9b47-592b4f45ee8a.png)